build.gradle

```diff
@@ -1,62 +1,58 @@
 import org.springframework.boot.loader.tools.LoaderImplementation
 
-buildscript {
-  repositories {
-    mavenLocal()
-    mavenCentral()
-    maven { url 'https://jitpack.io' }
-  }
-  dependencies {
-    classpath("net.serenity-bdd:serenity-gradle-plugin:2.4.34")
-  }
-}
-
 plugins {
     id 'application'
     id 'uk.gov.hmcts.java' version '0.12.63'
+    id 'idea'
     id 'jacoco'
     id 'pmd'
-    id 'com.github.ben-manes.versions' version '0.51.0'//do not change this
+    id 'com.github.ben-manes.versions' version '0.51.0'
     id "info.solidsoft.pitest" version '1.15.0'
     id 'io.spring.dependency-management' version '1.1.6'
-    id 'org.sonarqube' version '5.1.0.4882'
-    id 'org.springframework.boot' version '3.3.5'
+    id 'org.sonarqube' version '6.0.1.5171'
+    id 'org.springframework.boot' version '3.3.6'
     id "org.flywaydb.flyway" version '9.22.3'
-    id 'au.com.dius.pact' version '4.1.7' // do not change, otherwise serenity report fails
-    id 'org.owasp.dependencycheck' version '10.0.4'
+    id 'au.com.dius.pact' version '4.6.16'
+    id 'org.owasp.dependencycheck' version '11.1.1'
+    id 'net.serenity-bdd.serenity-gradle-plugin' version '4.2.8'
 }
 
-apply plugin: 'java'
-apply plugin: 'net.serenity-bdd.aggregator'
-apply plugin: 'project-report'
-apply plugin: 'idea'
-
 def versions = [
-  lombok             : '1.18.34',
-  gradlePitest       : '1.15.0',
-  pitest             : '1.16.1',
+  commonsCompress    : '1.27.1',
+  commonsIo          : '2.17.0',
```

Comment:
commons-io does not need to be downgraded, only ensured that version 2.11 is not being used

Reply:
Due to my fault build was failing and the error message was telling that it couldn't find 2.18.0 version. Attempted to downgrade to see if that helps only to realise the error.